src/roswire/distribution.py

```diff
@@ -2,7 +2,7 @@
 __all__ = ('ROSDistribution', 'ROSVersion')
 
 import enum
-from typing import Sequence
+from typing import Any, Sequence
 
 
 class ROSVersion(enum.IntEnum):
```

Comment:
Don't you need @total_ordering annotation?

Reply:
Yep :+1: